src/plugins/pcr_primer_design_for_dna_assembly/src/PCRPrimerDesignForDNAAssemblyOPWidget.cpp

```diff
@@ -380,6 +386,14 @@ void PCRPrimerDesignForDNAAssemblyOPWidget::sl_extractProduct() {
     }
 }
 
+void PCRPrimerDesignForDNAAssemblyOPWidget::sl_annotationCreationTaskFinished() {
+    CreateAnnotationsTask *t = qobject_cast<CreateAnnotationsTask *>(sender());
+    if (t->getState() != Task::State_Finished || t->isCanceled() || t->hasError()) {
+        return;
+    }
+    productsTable->setAnnotationGroup(t->getResultAnnotations().at(0)->getGroup());
```

Comment:
What if the 0 element isn't exist? It gotta be better to add SAFE_POINT, at least